jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-client/pom.xml

```diff
@@ -128,6 +118,37 @@
       <artifactId>org.eclipse.sisu.inject</artifactId>
       <scope>runtime</scope>
     </dependency>
+
+    <!-- Test deps -->
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.gwt.gwtmockito</groupId>
+      <artifactId>gwtmockito</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency><!-- to get CallerMock in unit test -->
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-testing-utils</artifactId>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <!-- because https://github.com/google/gwtmockito/issues/4 is fixed in 1.1.5 -->
+        <groupId>com.google.gwt.gwtmockito</groupId>
+        <artifactId>gwtmockito</artifactId>
+        <version>1.1.5</version>
+        <scope>test</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
```

Comment:
This override is no longer necessary. We recently upgraded to IP BOM 6.0.0.CR31 which contains the same version. https://github.com/jboss-integration/jboss-integration-platform-bom/blob/6.0.0.CR31/pom.xml#L134